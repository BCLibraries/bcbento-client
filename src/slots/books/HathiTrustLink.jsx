import React from 'react';

function HathiTrustLink({url}) {
    return <div className="hathi-trust-link">
        <div className="hathi-trust-link__icon">

        </div>
            <a href={url} target="_blank"  rel="noopener noreferrer" >
                Full Text Available at HathiTrust
                <i className="fa fa-external-link" aria-hidden="true"/>
            </a>
    </div>
}

export default HathiTrustLink;