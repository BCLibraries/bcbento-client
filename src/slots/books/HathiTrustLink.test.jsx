import renderer from 'react-test-renderer';
import HathiTrustLink from "./HathiTrustLink";

it('renders correctly', () => {
    const component = renderer.create(
        <HathiTrustLink url={'https://library.bc.edu'}/>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
