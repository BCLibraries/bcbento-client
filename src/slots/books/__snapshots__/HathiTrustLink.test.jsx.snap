// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="hathi-trust-link"
>
  <div
    className="hathi-trust-link__icon"
  />
  <a
    href="https://library.bc.edu"
    rel="noopener noreferrer"
    target="_blank"
  >
    Full Text Available at HathiTrust
    <i
      aria-hidden="true"
      className="fa fa-external-link"
    />
  </a>
</div>
`;
